.github/workflows/test.yml

```diff
@@ -3,18 +3,20 @@
 
 name: Python package
 
-on:
-  push:
-    branches: [ master ]
-  pull_request:
-    branches: [ master ]
+on: [push, pull_request]
```

Comment:
I dunno much about Github workflows, but does this change mean that any **untrusted code** that anybody pushes in a PR will get run automatically by GitHub CI? (and is that potentially problematic?)

Reply:
The only time this would make a difference in your project is if one of the contributors is developing on a branch in gpiozero/gpiozero - then the workflow would trigger on each push - probably a good thing.

It already triggers on PRs to master in the original version.

The advantage is that other developers working on a fork have the tests run in pipeline on each push to each branch - which is good practice. Leaving this out means anyone else who wants to make use of the workflow on a fork needs to make a similar change manually and then think to remove it before raising the PR.

Reply:
Apologies if I was being too cautious.